assets/storageclass_tier3.yaml

```diff
@@ -2,6 +2,8 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: tier3-csi
```

Comment:
can we rename this name to something more descriptive e.g: `ibm-powervs-tier3`

Reply:
Storage class names changed.